weasyl/oauth2.py

```diff
@@ -82,7 +106,9 @@ def POST(self):
         return body
 
 
-def get_userid_from_authorization(scopes=['wholesite']):
+def get_userid_from_authorization(scopes=None):
+    if not scopes:
+        scopes = ['wholesite']
     valid, request = server.verify_request(*(extract_params() + (scopes,)))
```

Comment:
This is confusing at this point for us. Can we do some more docstrings or comments?